spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -152,6 +152,21 @@ public void open() {
       }
     }
 
+    String remoteRepo = getProperty("zeppelin.interpreter.remoteRepo");
```

Comment:
was trying to fix CI error, not sure where to start...
Anyhow you are right, removing it.